cmd/agent/app/check.go

```diff
@@ -147,19 +148,7 @@ var checkCmd = &cobra.Command{
 			}
 
 			if check.IsJMXConfig(*conf) {
-				// we'll mimic the check command behavior with JMXFetch by running
-				// it with the JSON reporter and the list_with_metrics command.
-				fmt.Println("Please consider using the 'jmx' command instead of 'check jmx'")
-				if checkRate {
-					if err := RunJmxListWithRateMetrics(); err != nil {
-						return fmt.Errorf("while running the jmx check: %v", err)
-					}
-				} else {
-					if err := RunJmxListWithMetrics(); err != nil {
-						return fmt.Errorf("while running the jmx check: %v", err)
-					}
-				}
-
+				hasJMXConfig = true
```

Comment:
can you add a `FIXME` comment here that explains why we can't just run the JMX instances immediately here (because `RunJmxListWithMetrics` calls `common.SetupAutoConfig` a second time which breaks `rtloader`).

I'll add a card to our backlog so we address this.